src/keymanagement.c

```diff
@@ -200,6 +201,7 @@ PEP_STATUS validate_fpr(PEP_SESSION session,
             // delete key from being default key for all users/identities
             status = remove_fpr_as_default(session, fpr);
             // fallthrough intentional!
+        case PEP_ct_mistrusted:
```

Comment:
This function is used all over the place, and now has changed behavior, including DB writes. Why is this needed?

Reply:
You are right, I just made a make clean and make test and now some tests fail.
I think I need to rethink those changes and do this differently.
It was needed in order to set the key for signing the mail with an old identity, but I think I'll have to do this differently.